### PR TITLE
Add policy section

### PR DESCRIPTION
**Pull request by alvarosimon**
Replaces #457

### CODE REVIEW

File: mkdocs/docs/HPC/sites/hpc_policies.md [hunk 1]
```diff
@@ -1,5 +1,84 @@
-#HPC Policies
+# HPC Policies
 
 {% if site == gent %}
-Stub chapter
-{% endif %}
\ No newline at end of file
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
```
Comment (stdweird) [inline mkdocs/docs/HPC/sites/hpc_policies.md:4]: @epauwels you need to review this

File: config/templates/hpc.template [hunk 2]
```diff
@@ -96,6 +96,7 @@ theme:
     - navigation.expand
     # - navigation.indexes
     - navigation.tracking
+    - navigation.footer
```
[the remark this answers is not in the document]
Comment (alvarosimon) [reply config/templates/hpc.template:99]: @stdweird not sure why but the footer next/prev page links are missing in the current main, probably it was removed during the cleanup, it was there before. Nothing to do with the search.

**Comment by alvarosimon** [comment]
@epauwels I did a copy/paste from your original PR #457  but please check if everything is ok

File: config/templates/hpc.template (hunk 2 above)
[the remark this answers is not in the document]
Comment (alvarosimon) [reply config/templates/hpc.template:99]: @stdweird also included `search.share`  for the rest, it wouldn't do any harm

**Review by epauwels — APPROVED** [review]

**Pull request merged**